python/flare/shellcode_hash_search.py

```diff
@@ -342,6 +351,9 @@ def lookForOpArgs(self, start, end):
                     if t == idc.o_imm:
                         if using_ida7api:
                             opval = idc.get_operand_value(head, i)
+                            insn = idautils.DecodeInstruction(head)
+                            opmask = OPERAND_MASK[insn.ops[i].dtype]
```

Comment:
There are other possible dt_type values that we don't care about (dt_float, dt_double). Change to OPERAND_MASK.get() & check for None, in which case we skip masking.

Reply:
addressed in 083f743, thanks!